shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -317,6 +318,29 @@ - (NSOpenGLContext*)resourceContext {
   return _resourceContext;
 }
 
+- (float)displayRefreshRate {
```

Comment:
The style for the macOS embedding is to declare and document all private methods; see the top of this file.

Reply:
Will do.